lighthouse-core/audits/is-on-https.js

```diff
@@ -53,18 +64,16 @@ class HTTPS extends Audit {
           .filter(record => !HTTPS.isSecureRecord(record))
           .map(record => URL.elideDataURI(record.url));
 
+      const items = Array.from(new Set(insecureURLs)).map(url => ({url}));
```

Comment:
Moved this to before displayValue calc so that the display value # matches the de-duplicated item count.